distribution/lib/Standard/Base/0.0.0-dev/docs/api/Widget_Helpers.md

```diff
@@ -9,6 +9,6 @@
 - make_file_read_delimiter_selector -> Standard.Base.Any.Any
 - make_format_chooser include_number:Standard.Base.Data.Boolean.Boolean= include_date:Standard.Base.Data.Boolean.Boolean= include_date_time:Standard.Base.Data.Boolean.Boolean= include_time:Standard.Base.Data.Boolean.Boolean= include_boolean:Standard.Base.Data.Boolean.Boolean= -> Standard.Base.Any.Any
 - make_number_format_selector value:Standard.Base.Data.Numbers.Number= -> Standard.Base.Any.Any
-- make_regex_text_widget -> Standard.Base.Any.Any
+- make_regex_text_widget display:Standard.Base.Metadata.Display= -> Standard.Base.Any.Any
```

Comment:
I wonder if we could indicate the presence of `PRIVATE` tag?

From GUI perspective an API change of a fully visible component VS of a component that is exported but is not displayed in Component Browser, have slightly different 'severity' - so it could be easier to judge the changes.

Reply:
It is theoretically possible. However, I would like to integrate this PR as is, as soon as I fix all the tests. Let's keep your suggestions for some follow-ups.

Reply:
Sure!